docs/excel-example.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+title: Ingest Data to Excel
+permalink: /excelexample
+nav_order: 12
+---
+
+## Ingest WorkfloPlus Data to Excel    
+![WorkfloPlus Excel](assets/workfloplus-excel.png)
+The WorkfloPlus GraphQL Query API supports integration with many tools; one of the more common requests is for users to extract their data from WorkfloPlus into Excel.
+This example outlines an approach for not only ingesting data into Excel but moreover for creating a live link between Excel and WorkfloPlus so that the data set can be refreshed on demand from within Excel.
```

Comment:
Double newline to make space

Reply:
tried all sorts, doesn't seem to work between headers and images in the way that it does in body text